src/components/kpi/kpi.tmpl.html

```diff
@@ -20,11 +20,15 @@
           <h6>{{kpi.name.toUpperCase()}}</h6>
         </div>
         <alerts-config kpi="kpi" button-html="
-          <div class='im-button alert-caption'>
-            <span><a href='' ng-bind='kpi.layout.text.alert' /></span>
+          <div class='im-button alert-config-btn'>
+            <i class='fa fa-bell'></i>
```

Comment:
If this 'bell' button appears on the top of the kpi, I reckon it should be placed somewhere else, close to the "delete" button for example (instead of using 'absolute' + negative margin)